chrome/content/zotero/xpcom/proxy.js

```diff
@@ -69,6 +69,8 @@ Zotero.Proxies = new function() {
 		Zotero.Proxies.lastIPCheck = 0;
 		Zotero.Proxies.lastIPs = "";
 		Zotero.Proxies.disabledByDomain = false;
+
+		Zotero.Proxies.showRedirectNotification = Zotero.Prefs.get("proxies.showRedirectNotification");
```

Comment:
I was under the impression that [this line](https://github.com/LinuxMercedes/zotero/blob/optional-proxy-redirect-notification/chrome/content/zotero/preferences/preferences_proxies.js#L48) ensures that the variables set in the Proxies initialization are kept up to date with the settings page. 

I can change it so that it just checks the pref value directly, or, if this should really have a handler for pref updates, I'd be glad to set that up.